src/index.html

```diff
@@ -6,11 +6,45 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="form big"
+      action="https://mate.academy"
+      method="post"
+    >
+      <div class="container container-big">
+        <input
+          data-qa="keypress"
+          type="search"
+          class="search-bar search-bar-big"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
+    </form>
+    <form
+      data-qa="small"
+      class="form small"
+      action="https://mate.academy"
+      method="post"
```

Comment:
useless class `small`

Reply:
now I used it in stylesheet, .small::before selector